promptsource/templates/dynabench/dynasent/dynabench.dynasent.r2.all/templates.yaml

```diff
@@ -0,0 +1,93 @@
+dataset: dynabench/dynasent
+subset: dynabench.dynasent.r2.all
+templates:
+  0514a97d-0bbc-4f78-bcaa-622f2898fd8b: !Template
+    answer_choices: negative ||| positive ||| neutral ||| mixed
+    id: 0514a97d-0bbc-4f78-bcaa-622f2898fd8b
+    jinja: 'I''m reading a review that says "{{sentence}}".
+
+
+      Do you think the review is {{"positive"}} or {{"negative"}} or {{"neutral"}}
```

Comment:
order of responses in the answer choices doesn't match order of possible answers in the prompt. Please order so they are in the same order.